pprof/pprof.go

```diff
@@ -49,6 +52,7 @@ func GetRaw(opts Options) ([]byte, error) {
 // getArgs gets the arguments to run pprof with for a given set of Options.
 func getArgs(opts Options) ([]string, error) {
 	var pprofArgs []string
+	pprofArgs = append(pprofArgs, opts.ExtraArgs...)
```

Comment:
hmm. `pprofArgs` starts as an empty slice, shouldn't this just be:

```go
func getArgs(opts Options) ([]string, error) {
  pprofArgs := append([]string{}, opts.ExtraArgs...)
  // ...
```

assuming you actually want a copy, otherwise just reference ExtraArgs directly?